components/commands/browser/accelerator_pref_manager.cc

```diff
@@ -74,7 +75,9 @@ void AcceleratorPrefManager::AddAccelerator(
     const ui::Accelerator& accelerator) {
   ScopedDictPrefUpdate update(prefs_, kAcceleratorsPrefs);
   auto* list = update->EnsureList(base::NumberToString(command_id));
-  list->Append(ToCodesString(accelerator));
+  auto accelerator_string = ToCodesString(accelerator);
+  CHECK(!accelerator_string.empty());
```

Comment:
I don't know how I feel about the new guidance on `CHECK` versus `DCHECK`. It's pretty conceivable that we'll hit this (say, if the user has some funky keyboard which we don't handle). In that case, I think it's much better that we just don't add the shortcut than that we crash.

Alternatively, I guess it is technically a bug, so a crash report might help us fix it, but it doesn't seem great for the person trying to add the shortcut. WDYT?

Reply:
> I don't know how I feel about the new guidance on CHECK versus DCHECK

Yeah, me too 😭 It might be better to have DCHECK for DCHECK_IS_ON() and other reporting mechanism like base::DumplWithoutCrashing() for Release?